lib/packwerk/application_validator.rb

```diff
@@ -209,7 +209,7 @@ def check_inflection_file
     end
 
     def check_acyclic_graph
-      packages = Packwerk::PackageSet.load_all_from(".")
```

Comment:
`"."` resolves to the packwerk project's own directory, not the directory of the application under test.

Reply:
I think this bug fix could live in a separate PR. That way we can keep the scope of the PR a bit contained and merge the bug fix sooner.

Reply:
Definitely! None of this will be merged in the future PR that will introduce FixtureHelper.  The fixes you see in the app part of Packwerk came about while I was writing new unit tests, and found problems in the app itself.  Otherwise, I'd have to write unit tests that validate incorrect behaviour.